shoop/front/template_helpers/urls.py

```diff
@@ -31,8 +31,21 @@ def model_url(context, model, absolute=False):
 
 
 def has_url(url, *args, **kwargs):
+    """
+    Try to get the reversed URL for the given route name, args and kwargs.
+
+    If reverse resolution fails, returns None (instead of throwing an exception)..
+
+    :param url: URL name.
+    :type url: str
+    :param args: URL args
+    :type args: Iterable[object]
+    :param kwargs: URL kwargs
+    :type kwargs: dict[str, object]
+    :return: Reversed URL or None
+    :rtype: str|None
+    """
     try:
-        reverse(url, args=args, kwargs=kwargs)
-        return True
+        return reverse(url, args=args, kwargs=kwargs)
     except NoReverseMatch:
-        return False
+        return None
```

Comment:
has_url should probably be named get_url or something like that when it is used for getting the url.